test/functional/hostgroups_controller_test.rb

```diff
@@ -141,7 +141,13 @@ def setup_user(operation, type = 'hostgroups')
   end
 
   test "current_parameters should not fail on when hostgoup_parent_id parameter missing" do
-    xhr :post, :current_parameters, {:id => Hostgroup.first, :hostgroup => {}}, set_session_user
+    xhr :post, :current_parameters, {:hostgroup_id => Hostgroup.first, :hostgroup => {}}, set_session_user
```

Comment:
This should probably have hostgroup_parent_id in the request, but set to an empty string to match the UI.